data/graphics.js

```diff
@@ -9656,9 +9642,8 @@ var BattleMoveAnims = {
 			});
 			defender.delay(200);
 			defender.anim({
-				x: defender.leftof(+5),
```

Comment:
We already have a flinch animation (the ping thing) but I meant to add pokemon movement to it.